flights/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db   import models
+
+from core.models import TimeStampModel
+
+
+class FlightDetail(TimeStampModel): 
+    departure_time  = models.DateTimeField()
+    arrival_time    = models.DateTimeField()
+    remaining_seats = models.IntegerField()
+    reserved_seats  = models.IntegerField()
+    price           = models.DecimalField(max_digits=15, decimal_places=2)
+    flight_route    = models.ForeignKey('FlightRoute', on_delete=models.CASCADE)
+
+    class Meta: 
+        db_table    = 'flight_details'
+
+
+class FlightRoute(TimeStampModel): 
+    code            = models.CharField(max_length = 10)
+    departure       = models.ForeignKey('Location', related_name = 'departure', on_delete = models.CASCADE)
+    destination     = models.ForeignKey('Location', related_name = 'destination', on_delete = models.CASCADE)
+    airpalne        = models.ForeignKey('Airplane', on_delete = models.CASCADE)
```

Comment:
1. = 기준으로 정렬에 너무 많은 공백이 있습니다.
2. departure와 destination의 related_name을 주셨는데, related_name을 사용하여 데이터를 가져올 때 어떤 데이터가 오는지 생각해보고 수정해 보시면 좋을 것 같아요
```suggestion
    code        = models.CharField(max_length = 10)
    departure   = models.ForeignKey('Location', related_name = 'departure', on_delete = models.CASCADE)
    destination = models.ForeignKey('Location', related_name = 'destination', on_delete = models.CASCADE)
    airpalne    = models.ForeignKey('Airplane', on_delete = models.CASCADE)
```

Reply:
출발지와 도착지 id(pk)값이 오는 컬럼인데, departure_id, destination_id로 바꾸면 될까요?

Reply:
related name을 안쓰면 자동으로 departure_id, destination_id로 지정되는걸까요 한번 해보겠습니다~